src/app/drivingLicence/controllers/root.js

```diff
@@ -19,10 +18,9 @@ class RootController extends BaseController {
 
     if (process.env.AUTH_SOURCE_ENABLED === "true") {
       //axios.post to new personInfo endpoint, put licence details into req.session.shared_claims shared claims
-      const personInfoApiResponse = await req.axios.get(
-        `${BASE_URL}/${PERSON_INFO}`,
-        { headers: headers }
-      );
+      const personInfoApiResponse = await req.axios.get(`${PERSON_INFO}`, {
```

Comment:
We need the BASE_URL in here to build the full path to reach the personInfo endpoint so can we add that back in? Unless I'm missing something here and the baseUrl is implicitly added by the configuration of axios somewhere?